packages/tsconfig-reference/scripts/types.ts

```diff
@@ -27,7 +27,7 @@ export interface DiagnosticMessage {
   message: string;
   reportsUnnecessary?: {};
   /* @internal */
-  elidedInCompatabilityPyramid?: boolean;
+  elidedInCompatibilityPyramid?: boolean;
```

Comment:
```suggestion
  elidedInCompatabilityPyramid?: boolean;
```

I'm afraid this typo exists in TypeScript itself, so we can't actually change that here.